### DISCUSSION TITLE
Upgrade to Bootstrap 5.x.x

### DISCUSSION THREAD

**Discussion by m1chale** [pretix/pretix#3752] [General]
Are there any plans on updating the Bootstrap version?
The currently used version (v3.x.x) has reached the end-of-life a long time ago.
https://endoflife.date/bootstrap

If pretix will be used in environments that require pentesting (for me that is the case), this will cause the test to fail as there are no more critical/security updates.

**Comment by raphaelm** [comment]
There are currently no plans to ever do that, as the effort would be inconceivably gigantic for very little gain. After all, it's mostly a CSS - framework and CSS frameworks don't have security issues. Yes, it includes a bunch of JS, only very little of which we use, and we are not aware of security issues relevant to the usage in pretix. If your pentest process cannot account for that, I'm sorry 😔

I think we'll most likely over time migrate to something self-made that builds up on bootstrap3 but modernizes individual aspects at some point.

**Reply by m1chale** [reply]
Hey @raphaelm, thanks for your very quick response! I fully agree on the CSS-Framework part (unfortunately, not everyone else does - maybe a point for discussion :D) and I get your point regarding the effort for an upgrade.

Anyways, can I ask you one more thing regarding dependencies? Is there a chance that pretix will get an update on moment.js and moment-timezone.js, as the currently used versions come with some vulnerabilities?

[CVE-2022-24785](https://nvd.nist.gov/vuln/detail/CVE-2022-24785) 
[CVE-2022-31129](https://nvd.nist.gov/vuln/detail/CVE-2022-31129)

That fix would just require a tiny version bump.
